Tom-tom7m/Homework-3/object-homework/app.js

```diff
@@ -0,0 +1,103 @@
+"use strict";
+
+/////////////////////////////////////////////////////////////
+
+///////////////////////// Challenge /////////////////////////
+
+/////////////////////////////////////////////////////////////
+
+// Write your answers inside this file, at the places where it's indicated by the comments.
+
+
+// 1. Suppose that we wanted to create a browser-based calendar program for keeping track of things. In comments, list at least three reasonable abstractions that you might use to build this program; for each abstraction, list out at least two properties and behaviors that it would make sense for that abstraction to have. Below is an example - please write your answer in the same format.
+  // 'Car'
+  // Description: Our app manages car sales.
+  // Every 'car' has
+  // - a make
+  // - a model
+  // - a year
+  // Every 'car' can
+  // - drive
+  // - brake
+  // - park
+
+// Answer Starts Here
+
+/* 'Calendar Program'
+      Description: Our app manages a personal calendar / to-do list.
+        Every 'Calendar Program' has
+          - calendar / date-picker
+          - to-do list tracker
+        Every 'Calendar Program' can
+          - link to-do items with your personal calendar
+          - allow image uploads for a particular to-do item
+          - send reminders for upcoming to-do items
+          - upload to your personal calendar
+        Every 'Calendar Program' accepts
+          - user input for new to-do items
+          - new calendar events from your personal linked calendar
+*/
+
+// Answer Ends Here
+
+
+// 2. Create an Object literal that lines up with the following description. Store it in the variable 'pet_owner', below. Be sure to give it reasonable values for each of its properties.
+
+  // 'Owner'
+  // Description: We are making an app for a veterinary clinic - it manages pets' vaccinations.
+  // Every 'owner' has:
+  // - a name
+  // - an address
+
+let pet_owner = {};
+
+// Answer Starts Here
+
+let ownerName, address;
+
+pet_owner.ownerName = 'Tom';
+pet_owner.address = '1122 Tennessee Street, San Francisco, 94107';
+
+console.log(pet_owner);
+
+// Answer Ends Here
+
+
+// 3. Create an Object literal that lines up with the following description. Store it in the variable `some_pet`, below.
+
+  // Pet
+  // Description: We are making an app for a veterinary clinic - it manages pets' vaccinations.
+  // Every 'pet' has:
+  // - a name
+  // - a species
+  // - a breed
+  // - a noise that it can make (e.g. 'bark', 'meow', etc.)
+  // Every pet can:
+  // - make noise (each pet makes its own unique noise, as specified by `noise`.
+
+let some_pet;
+
+// Answer Starts Here
+
+let petName, species, breed, noise;
+
+some_pet = {
+  petName: "Mozzarella",
+  species: "Dog",
+  breed: "Cavalier King Charles Spaniel",
+  noise: "Bark",
+  makeNoise: function() {
+    if(some_pet.noise){"Yes";
```

Comment:
We need a return statement or a console.log statement in order to send the result back to the user calling the function